pkgs/applications/networking/sniffers/savvycan/default.nix

```
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, qtbase
+, qttools
+, qmake
+, qtserialbus
+, qtserialport
+, qtdeclarative
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "savvycan";
+  version = "208";
+
+  src = fetchFromGitHub {
+    owner = "collin80";
+    repo = "SavvyCAN";
+    rev = "V${version}";
+    sha256 = "sha256-agvCl8c7LqGyIKe0K3PdzuBUqTJZtUr434134olbUMw=";
```

You're using a SRI hash, so:

```suggestion
    hash = "sha256-agvCl8c7LqGyIKe0K3PdzuBUqTJZtUr434134olbUMw=";
```